### PR TITLE
Correct formatting --arguments

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
Fix a few formatting issues

##### ISSUE TYPE
 - Docs Pull Request

### CODE REVIEW

File: docs/docsite/rst/vault.rst [hunk 1]
```diff
@@ -14,12 +14,12 @@ For best practices advice, refer to :ref:`best_practices_for_variables_and_vault
 What Can Be Encrypted With Vault
 ````````````````````````````````
 
-The vault feature can encrypt any structured data file used by Ansible.  This can include "group_vars/" or "host_vars/" inventory variables, variables loaded by "include_vars" or "vars_files", or variable files passed on the ansible-playbook command line with "-e @file.yml" or "-e @file.json".  Role variables and defaults are also included!
+The vault feature can encrypt any structured data file used by Ansible.  This can include "group_vars/" or "host_vars/" inventory variables, variables loaded by "include_vars" or "vars_files", or variable files passed on the ansible-playbook command line with ``-e @file.yml`` or ``-e @file.json``.  Role variables and defaults are also included!
 
-Ansible tasks, handlers, and so on are also data so these can be encrypted with vault as well. To hide the names of variables that you're using, you can encrypt the task files in their entirety. However, that might be a little too much and could annoy your coworkers :)
```
Comment (gundalow) [inline docs/docsite/rst/vault.rst]: Removed `However, that might be a little too much and could annoy your coworkers :)`

File: docs/docsite/rst/playbooks_vault.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@ Using Vault in playbooks
 
 New in Ansible 1.5, "Vault" is a feature of ansible that allows keeping sensitive data such as passwords or keys in encrypted files, rather than as plaintext in your playbooks or roles. These vault files can then be distributed or placed in source control.
 
-To enable this feature, a command line tool, `ansible-vault` is used to edit files, and a command line flag `--ask-vault-pass` or `--vault-password-file` is used. Alternately, you may specify the location of a password file or command Ansible to always prompt for the password in your ansible.cfg file. These options require no command line flag usage.
+To enable this feature, a command line tool, ``ansible-vault`` is used to edit files, and a command line flag ``--ask-vault-pass`` or ``--vault-password-file`` is used. Alternately, you may specify the location of a password file or command Ansible to always prompt for the password in your ``ansible.cfg file``. These options require no command line flag usage.
```
Comment (alikins) [inline docs/docsite/rst/playbooks_vault.rst]: Could these use the ':option:' link? That shoud point the link at the cli command docs page.
(And a ':ref:' for 'ansible-vault'...)

rst/vault.rst uses that style for ex.

``` rst
:option:`--ask-vault-pass <ansible-vault --ask-vault-pass>` and 

To enable this feature, a command line tool, :ref:`ansible-vault` is used to edit files, and a command line flag 
:option:`--ask-vault-pass <ansible-vault --ask-vault-pass>`  or :option:`--vault-password-file <ansible-vault --vault-password-file>` is used.

```

Comment (gundalow) [reply docs/docsite/rst/playbooks_vault.rst]: Good idea, done

Comment (gundalow) [reply docs/docsite/rst/playbooks_vault.rst]: Done

**Comment by gundalow** [comment]
ready_for_review

File: docs/docsite/rst/playbooks_vault.rst [hunk 3]
```diff
@@ -35,7 +35,7 @@ If you are using a script instead of a flat file, ensure that it is marked as ex
 
 This is something you may wish to do if using Ansible from a continuous integration system like Jenkins.
 
-(The ``--vault-password-file`` option can also be used with the :ref:`ansible-pull` command if you wish, though this would require distributing the keys to your nodes, so understand the implications -- vault is more intended for push mode).
+The :option:`--vault-password-file <ansible-vault --vault-password-file>` option can also be used with the :ref:`ansible-pull` command if you wish, though this would require distributing the keys to your nodes, so understand the implications -- vault is more intended for push mode.
```
Comment (alikins) [inline docs/docsite/rst/playbooks_vault.rst]: doc refernces ansible-pull, so the :option: link should reference the 'ansible-pull' program so it links to http://docs.ansible.com/ansible/devel/ansible-pull.html#cmdoption-ansible-pull-ask-vault-pass

Something like:


```
The :option:`--vault-password-file <ansible-pull --vault-password-file>` option 
```

**Review by alikins — CHANGES_REQUESTED** [review]
fix ansible-pull option link

**Review by alikins — APPROVED** [review]

**Review by dharmabumstead — APPROVED** [review]
Good to go with some mild edits. Thanks @gundalow.

**Comment by gundalow** [comment]
Adding to `2.4` project so we can benefit from better docs

**Comment by abadger** [comment]
cherry-picked for 2.4.2beta1 release.

**Pull request merged**